images/dcos/bootstrap.sh

```diff
@@ -194,11 +199,27 @@ ${apply_uids}
   --address=${host_ip}
   --cloud-config=${cloud_config}
   --cloud-provider=mesos
-  --master=http://${host_ip}:${apiserver_port}
+  --master=${host_kube_master}
```

Comment:
can we just have this use the same value as the scheduler and get rid of this envvar?